src/slice.rs

```diff
@@ -453,6 +502,12 @@ impl<D1: Dimension, T> SliceNextDim<D1, D1::Larger> for Range<T> {
     }
 }
 
+impl<D1: Dimension, T> SliceNextDim<D1, D1::Larger> for RangeInclusive<T> {
+    fn next_dim(&self, _: PhantomData<D1>) -> PhantomData<D1::Larger> {
+        PhantomData
```

Comment:
Another macro question :) From line 499 to 527, would it be possible to create a macro for that? `next_dim` is implemented for all types of `Range` except `RangeFull`.

Reply:
Yes, it is a lot more concise with a macro. Thanks!